gaurun/notification.go

```diff
@@ -96,6 +96,9 @@ func pushNotificationIos(req RequestGaurunNotification) error {
 	token := req.Tokens[0]
 
 	headers := NewApnsHeadersHttp2(&req)
+	if APNSClient.Token != nil {
+		headers.AuthToken = APNSClient.Token
+	}
```

Comment:
This process looks to be unclear. Instead, I think it's good to create `NewApnsHeaderHttp2WithToken()`.

Reply:
I added `NewApnsHeaderHttp2WithToken`.